sql/core/pom.xml

```diff
@@ -69,6 +69,11 @@
       <version>2.3.0</version>
     </dependency>
     <dependency>
+      <groupId>org.jodd</groupId>
+      <artifactId>jodd-core</artifactId>
+      <version>${jodd.version}</version>
+    </dependency>
```

Comment:
I have read the documents of joda-time. The toJulianDayNumber API only valid since 2.2, but what we use in spark is 2.1.